proxy/src/main/java/org/apache/rocketmq/proxy/common/MessageReceiptHandle.java

```diff
@@ -27,24 +29,26 @@ public class MessageReceiptHandle {
     private final String messageId;
     private final long queueOffset;
     private final String originalReceiptHandle;
-    private final long timestamp;
     private final int reconsumeTimes;
-    private final long expectInvisibleTime;
 
-    private String receiptHandle;
+    private final AtomicInteger renewRetryTimes = new AtomicInteger(0);
+    private long timestamp;
+    private long expectInvisibleTime;
+    private volatile String receiptHandle;
 
-    public MessageReceiptHandle(String group, String topic, int queueId, String receiptHandle, String messageId,
-        long queueOffset, int reconsumeTimes, long expectInvisibleTime) {
+    public MessageReceiptHandle(String group, String topic, int queueId, String receiptHandleStr, String messageId,
+        long queueOffset, int reconsumeTimes) {
+        ReceiptHandle receiptHandle = ReceiptHandle.decode(receiptHandleStr);
         this.group = group;
         this.topic = topic;
         this.queueId = queueId;
-        this.receiptHandle = receiptHandle;
-        this.originalReceiptHandle = receiptHandle;
+        this.receiptHandle = receiptHandleStr;
```

Comment:
Can here rename to receiptHandleStr for avoiding confusion with ReceiptHandle receiptHandle

Reply:
done